tests/tests_unit/test_data_classes/test_data_models/test_data_types.py

```diff
@@ -37,12 +37,28 @@ class TestPropertyType:
                 "container": {"space": "mySpace", "externalId": "myId", "type": "container"},
                 "list": True,
             },
+            {
+                "type": "enum",
+                "values": {
+                    "string": {
+                        "name": "string",
+                        "description": "Time series with string data points.",
+                    },
+                    "numeric": {
+                        "name": "numeric",
+                        "description": "Time series with double floating point data points.",
+                    },
+                },
+            },
         ],
+        ids=lambda d: d.get("type", "unknown"),
```

Comment:
Just for nice info for IDE when running tests:
![image](https://github.com/user-attachments/assets/0ae3c597-b788-45d8-91da-977be93d217e)